src/controllers/userController.js

```diff
@@ -60,12 +65,27 @@ export const registerUser = async (req, res) => {
     user.password = await BcryptUtility.hashPassword(req.body.password);
     const { id, email } = await UserService.register(user);
     const userData = { id, email };
+    console.log(userData);
     const userToken = await generateToken(userData);
+    const message = {
+      from: `"Falcons Project" <${process.env.FROM_EMAIL}>`,
+      to: userData.email,
+      subject: "Falcons project - Verify your account",
+      text: `Hello, Thank you for registering on our site. Please click on this link to verify your email address: http://${process.env.url}/verify-account?${userToken}. If you did not register for an account with Falcons Project, please ignore this email.`,
+      html: `
+    <h1> Hello,</h1>
+    <p> Thanks for registering on our site.</p>
+    <p> Please click on the button below to verify your account.</p>
+    <a href="${process.env.url}/verify-account?${userToken}" style="background-color:#008CBA;color:#fff;padding:14px 25px;text-align:center;text-decoration:none;display:inline-block;border-radius:4px;font-size:16px;margin-top:20px;">Verify Account</a>
+    <p>If you did not register for an account with Falcons Project, please ignore this email.</p>`,
+    };
+    await sgMail.send(message);
+
```

Comment:
why can't use this as helper function because we will need to send email at any time use it and I think it's here